src/common/lib/client/defaultrest.ts

```diff
@@ -0,0 +1,24 @@
+import Platform from 'common/platform';
+import ClientOptions from 'common/types/ClientOptions';
+import Message from '../types/message';
+import PresenceMessage from '../types/presencemessage';
+import { BaseClient } from './baseclient';
+import Rest from './rest';
+
+/**
+ * Preloaded BaseClient with all REST features, used as the default non-treeshakeable Rest export
+ */
+class DefaultRest extends BaseClient {
+  constructor(options: string | ClientOptions) {
+    super(options, {
+      Rest,
+    });
+  }
+
+  static Platform = Platform;
```

Comment:
Shouldn't these static properties all go on `BaseClient`? They're part of the API that it's meant to satisfy.

Reply:
(and `BaseClient` already has a static `Platform` property)

Reply:
i've removed `Platform` here though, nice spot 👍